<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeightDialog</class>
 <widget class="QMainWindow" name="WeightDialog">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QLineEdit" name="weightInput"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Weight: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="edgeSelect"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButtonOK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Edge: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit application (Q)</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionVisToggleInput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>I</string>
   </property>
   <property name="toolTip">
    <string>Toggle Input (I)</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>resize</string>
   </property>
   <property name="toolTip">
    <string>reset Zoom (R)</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionToggleFullscreen">
   <property name="text">
    <string>F</string>
   </property>
   <property name="toolTip">
    <string>toggle fullscreen (F)</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionVisToggleInputLabels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>L</string>
   </property>
   <property name="toolTip">
    <string>Toggle Input Label Visibility (L)</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionVisToggleArcs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S</string>
   </property>
   <property name="toolTip">
    <string>Toggle Straight-Skeleton Arcs Visibility (S)</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionTimeForwardAfterChains">
   <property name="text">
    <string>C</string>
   </property>
   <property name="toolTip">
    <string>finsh computing both chains (C)</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionFinishComputation">
   <property name="text">
    <string>‚èé</string>
   </property>
   <property name="toolTip">
    <string>compute final skeleton (Return)</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionDefineWeight">
   <property name="text">
    <string>W</string>
   </property>
   <property name="toolTip">
    <string>define a weight for one edge (W)</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionEventStep">
   <property name="text">
    <string>n</string>
   </property>
   <property name="toolTip">
    <string>compute next event (N)</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
